_submissions/round-12/1/2015-02-24-tracking-changes-to-files.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2015-02-24
+round: Round 12
+title: Tracking changes to files
+author: April Clyburne-Sherin
+permalink: /2015/02/tracking-changes-to-files/
+tags:
+  - Git
+  - Changes
+  - Tracking
+---
+
+My concept map covers the concepts covered in the Software Carpentry lesson ["Tracking changes to files"](http://swcarpentry.github.io/git-novice/01-backup.html).
+
+The lesson covers how to check the status of files in the repository, how to add them to be tracked by git, how to commit them, and how to check the log of revisions.
+
+Please review my concept map and help me improve it:
+
+[Tracking changes to files](http://imgur.com/tkC9uDy)
```

Comment:
Please change the link to `http://i.imgur.com/tkC9uDy.jpg` - what you have links to the imgur page, not to the image itself.
Thanks,
Greg